build.gradle

```diff
@@ -16,9 +16,6 @@ sourceSets {
             srcDir 'javarosa/util/schema-gen/src'
             compileClasspath += main.output
         }
-        resources{
-            srcDir 'javarosa/src/test/resources'
-        }
```

Comment:
I'm fairly certain that the schema-gen code doesn't need to point to these test resources. Does that sound right?